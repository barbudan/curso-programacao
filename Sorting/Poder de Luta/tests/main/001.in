3
Yamcha 1289
Tenshinhan 2500
Chichi 5212