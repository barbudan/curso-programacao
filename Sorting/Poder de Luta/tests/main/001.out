Chichi 5212
Tenshinhan 2500
Yamcha 1289
